### DISCUSSION TITLE
Question on getting date automatically filled in Daily note

### DISCUSSION THREAD

**Discussion by jasmeetchadha** [chhoumann/quickadd#131] [Q&A] [answered]
Hi - I use QuickAdd to add data into my daily note and when that doesnt exist create it. 

In my template for Daily Note, I have the following fields
Summary::
Date:: {{Date:YYYY-MM-DD}}
Context: 
People:

I am hoping to get the date stamped on the day I create the daily using current date. 

Unfortunately, I dont get the result with the above code? How should I do it?

**Comment by chhoumann** [comment, marked as answer]
Format syntax needs to be capitalized - e.g. {{DATE:YYYY-MM-DD}}

You can see more here:
https://github.com/chhoumann/quickadd/blob/master/docs/FormatSyntax.md

**Reply by jasmeetchadha** [reply]
That's perfect. It needed to be CAPS for DATE. Thanks
